community/people/SH/index.md

```diff
@@ -0,0 +1,25 @@
+---
+Layout: Person
```

Comment:
I think these all need to be lower case `layout`, etc...  The data is not being processed properly with the capitalized version of these tags

Reply:
okay, made sure I changed to lower case